seneca.js

```diff
@@ -721,23 +728,27 @@ function make_seneca (initial_options) {
     var pattern_rules = _.clone(action.validate || {})
     _.each(pattern, function (v, k) {
       if (_.isObject(v)) {
-        pattern_rules[k] = v
+        pattern_rules[k] = _.clone(v)
         delete pattern[k]
       }
     })
 
-    actmeta.rules = pattern_rules
-
     var addroute = true
 
+    // TODO: deprecate
```

Comment:
can we add these information in the code, because it is really hard to understand one of those `TODO: deprecate` without any notes about it.